server/src/main/java/org/elasticsearch/script/StoredScriptSource.java

```diff
@@ -481,7 +481,9 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
         builder.startObject();
         builder.field(LANG_PARSE_FIELD.getPreferredName(), lang);
         builder.field(SOURCE_PARSE_FIELD.getPreferredName(), source);
-        builder.field(OPTIONS_PARSE_FIELD.getPreferredName(), options);
+        if (options.isEmpty() == false) {
+            builder.field(OPTIONS_PARSE_FIELD.getPreferredName(), options);
+        }
```

Comment:
why do we need this change? it seems like it changes the existing StoredScriptSource output, that may be risky to do, not sure.

Reply:
there is a custom serialization performed in `RestGetStoredScriptAction` - I use a standard  `StoredScriptSource.toXContent` but it violates test `rest-api-spec/test/painless/16_update2.yml` - it expects that empty `options` are not shows at all. Deserialization of empty `options` or no-options are equal - resolves to empty map.

I traced call sites - seems no any other usages of `StoredScriptSource.toXContent` + other tests passes fine.

Reply:
ok thanks